web_m2x_options/static/description/index.html

```diff
@@ -0,0 +1,412 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Can you remove this line please ?

It raises a warning
`DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, web_m2x_options can just have a UTF8 description with not need for a declaration.`

Reply:
I have deleted the line